bears/c_languages/GNUIndentBear.py

```diff
@@ -152,16 +154,16 @@ def create_arguments(filename, file, config_file,
             Uses Kernighan & Ritchie coding style.
         :param linux_style:
             Uses Linux coding style.
-        :param tab_width:
-            Number of spaces per indent level.
+        :param indent_size:
+            Number of spaces representing one tab.
```

Comment:
number of spaces representing one tab? that's better termed as ``tab_width`` or ``tab_size`` right?